config/runtime.exs

```diff
@@ -0,0 +1,13 @@
+import Config
+
+config :geo,
+  disable_refresh: true
```

Comment:
Be careful about this being in production, this will stop the database from refreshing.